deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -764,11 +771,15 @@ <h4 class="panel-title pull-left pointer-cursor">
                                     function (item) {
                                         return {
                                             value: item.id,
-                                            text: item.provider_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.provider + ": " + item.abstract_name + ")",
-                                            isSelected: item.id === data[0].id
+                                            text: item.provider_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.network + ", " + item.provider + ": " + item.abstract_name + ")",
+                                            isSelected: item.id === data[0],
+                                            isDisabled: item.retired
                                         }
                                     });
                             capacitySetting.selectedHostTypeValue = data[0].id;
+                            setTimeout(function() {
+                                $(".single-select-search").trigger("chosen:updated");
```

Comment:
Looks like this happens in the background with 50ms delay. Is that expected? Could be worth having an explanation in a comment similar to: https://github.com/pinterest/teletraan/blob/468c0653600e158c6e6ebadf3771efb4b2e6412e/deploy-board/deploy_board/templates/clusters/cluster_configuration.html#L1024-L1029

Reply:
Could `Vue.nextTick` be used to be consistent with the other code?

Reply:
Yes that seems to work. I will update it.